### PR TITLE
Create roadmap-item.md

### PR DESCRIPTION
**Pull request by vincent6767**
### Description
The context of the roadmap item can be read here: o3de/sig-release#79.

### How was this PR tested?

I merged this PR in the SIG release and the "Roadmap Item" shows as expected
image

### Action Needed
Since the RFC leverage three labels, kind/roadmap, needs-triage, and triage/accepted, please create this label in your SIG repo if it does not exist already once you merge the PR.

### needs-triage
Taken from [O3DE label](https://github.com/o3de/o3de/labels?q=needs-triage)
- label name: needs-triage
- description: Indicates an issue or PR lacks a triage/foo label and requires one

### triage/accepted
Taken from [O3DE Label](https://github.com/o3de/o3de/labels?q=triage%2Faccepted)
- label name: triage/accepted
- description: Indicates an issue or PR is ready to be actively worked on.

### kind/roadmap
Taken from [O3DE Label](https://github.com/o3de/o3de/labels?q=kind%2Froadmap)
- label name: **kind/roadmap**
- description: **Categorizes an issue that goes in the O3DE Public Roadmap**

Signed-off-by: Vincent <vincentbiasa6767@gmail.com>

### CODE REVIEW

**Comment by bhanuja-s** [comment]
Hi @vincent6767, just trying to understand the context here after a long break. Would you like us to review this template, or would you like us (as representatives of sig-ui-ux) to implement this in our repository - o3de/sig-ui-ux?

**Comment by vincent6767** [comment]
Hi @bhanuja-s , both actually. We want SIG UI/UX to implement this in the repo as part of https://github.com/o3de/sig-release/issues/79, but at the same time feel free to leave any comments/feedback if you have any.

**Comment by vincent6767** [comment]
@bhanuja-s @rainbj Please help review!

**Comment by bhanuja-s** [comment]
Ah apologies for the late response, I'm happy to accept this into our SIG repo and don't have any further feedback as this looks clear!

**Review by bhanuja-s — APPROVED** [review]
Approved!

**Comment by vincent6767** [comment]
Thanks! @bhanuja-s Do we need to wait for @rainbj to approve before merging?

**Comment by vincent6767** [comment]
Merged. Thanks @bhanuja-s @rainbj !

**Comment by rainbj** [comment]
Thanks for reviewing this @bhanuja-s

**Pull request merged**
